wbooks/src/interfaces/redux.ts

```diff
@@ -1,7 +1,5 @@
 export interface BookState {
   books: [];
-  isLoading: boolean;
-  hasError: boolean;
```

Comment:
if you use r-r completer, you would have `[field]Loading` (booelean) and `[field]Error` (response.error)

```
booksLoading: boolean;
booksError:  string;  ?)
```